Cargo.toml

```diff
@@ -161,6 +161,7 @@ fc-rpc-core = { version = "1.1.0-dev", git = "https://github.com/AstarNetwork/fr
 
 # Frontier Primitive
 fc-storage = { version = "1.0.0-dev", git = "https://github.com/AstarNetwork/frontier.git", branch = "polkadot-v0.9.40", default-features = false }
+fp-account = { version = "1.0.0-dev", git = "https://github.com/AstarNetwork/frontier.git", branch = "polkadot-v0.9.40", default-features = false }
```

Comment:
@luckychacha Why do we enclude this from the outside repo. Can we for it maybe? @xcthulhu What do you think? Otherwise we will have a potential security hole if our installation depends on third part githubs

Reply:
1. The dependency `fp-account` in my requirements is used to build the mock runtime for benchmark's unit tests, and thus is placed under `[dev-dependencies]` in the `runtime-common` module. It will not be included in the release version. Moreover, I have found alternative ways to build the mock runtime, so I will remove the `fp-account.workspace = true` entry from the `runtime-common`'s `Cargo.toml` file under `[dev-dependencies]`.
2. I have noticed that our cargo.toml file contains a patch for this dependency, pointing to our forked `frontier` project:
   ```toml
   # Temporary, leaks the std feature in the non-std env. Until https://github.com/AstarNetwork/frontier/pull/84 is merged.
[patch."https://github.com/AstarNetwork/frontier.git"]
fp-account = { git = "https://github.com/GoldenGateGGX/frontier.git", branch = "polkadot-v0.9.40" }
   ```
   This patch ensures that we are actually utilizing our own forked version of the `frontier` project rather than the one under `AstarNetwork`.